draft-ietf-mmusic-ice-sip-sdp-36.xml

```diff
@@ -473,7 +483,10 @@ An agent sets the rest of the ICE-related fields in the SDP for this data stream
 would in an initial offer of this data stream (see <xref target="sec-encoding"/>).
 Consequently, the set of candidates MAY include some, none, or all of the
 previous candidates for that data stream and MAY include a totally new set of
-candidates.
+candidates. The agent MAY modify the attribute values of the SDP ice-options and
+SDP ice-pacing attributes, and it MAY change its role using the SDP ice-lite attribute.
+The agent MUST NOT modify the SDP ice-options, ice-pacing and ice-lite attributes in a
+subsequent offer unless the offer is sent in order to request an ICE restart.
 </t>
```

Comment:
might want to add reference to ICE restart section here as well for uniformity

Reply:
The text is in the ICE Restart section.

Reply:
I added a reference in the answerer procedures for ICE restarts.